### ISSUE TITLE
Can not import image 

### ISSUE THREAD

**Issue by junaid18183** [weaveworks/ignite#130]
The import image command is failing. 

```
# ignite image import weaveworks/ignite-ubuntu:latest
INFO[0000] Starting image import...
failed to format image 5f3dc10d89792555: command ["mkfs.ext4" "-I" "256" "-E" "lazy_itable_init=0,lazy_journal_init=0" "/var/lib/firecracker/image/5f3dc10d89792555/image.ext4"] exited with "mke2fs 1.42.9 (28-Dec-2013)\n/var/lib/firecracker/image/5f3dc10d89792555/image.ext4 is not a block special device.\nProceed anyway? (y,n) ": exit status 1

ignite version - 
# ignite version
Ignite version: version.Info{Major:"0", Minor:"4", GitVersion:"v0.4.0", GitCommit:"e0e7e8c50cd3c6532486625393dff4d415081829", GitTreeState:"clean", BuildDate:"2019-07-10T12:30:11Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
Firecracker version: v0.17.0

OS and kernal  version - 
# cat /etc/redhat-release
Red Hat Enterprise Linux Server release 7.6 (Maipo)
# uname -a
Linux ip-172-31-4-245.us-gov-west-1.compute.internal 3.10.0-957.10.1.el7.x86_64 #1 SMP Thu Feb 7 07:12:53 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux```

**Comment by junaid18183** [comment]
Manual running the command fails with error invalid blocks 
```[root@ip-172-31-4-245 ~]# mkfs.ext4 -I 256 -F lazy_itable_init=0,lazy_journal_init=0 /var/lib/firecracker/image/5f3dc10d89792555/image.ext4
mke2fs 1.42.9 (28-Dec-2013)
mkfs.ext4: invalid blocks '/var/lib/firecracker/image/5f3dc10d89792555/image.ext4' on device 'lazy_itable_init=0,lazy_journal_init=0'```

**Comment by junaid18183** [comment]
created a PR - https://github.com/weaveworks/ignite/pull/131

**Comment by jiangpengcheng** [comment]
met the same problem

**Comment by luxas** [comment]
Hi @junaid18183 and @jiangpengcheng :wave:!
Thanks for trying Ignite out, and for the PR :smile:
It's true, we haven't tested it on RHEL, so this needs to get fixed.

Will check out the PR, and get this fix into v0.4.1

**Comment by luxas** [comment]
Fixed with https://github.com/weaveworks/ignite/pull/131 :tada:
